definitions/procedures/backup/metadata.rb

```diff
@@ -6,6 +6,7 @@ class Metadata < ForemanMaintain::Procedure
       preparation_steps { Checks::Foreman::DBUp.new if feature(:foreman_server) }
       param :backup_dir, 'Directory where to backup to', :required => true
       param :incremental_dir, 'Changes since specified backup only'
+      param :online_backup, 'Selected online backup ?'
```

Comment:
As this param seems to be boolean type then would it be possible to declare `:online_backup` as a flag?

```suggestion
     param :online_backup, 'Select for an online backup',
            :flag => true, :default => false
```